src/guides/v2.3/install-gde/prereq/apache.md

```diff
@@ -196,11 +188,94 @@ Installing and configuring Apache is basically a three-step process: install the
 1. Continue with the next section.
 
    {:.bs-callout-info}
-   Even though Apache 2.4 is provided by default with CentOS 7, you configure it like Apache 2.2. See the following section.
+   Even though Apache 2.4 is provided by default with CentOS 7. See the following section to configure it.
+
+### Enable rewrites and .htaccess for CentOS 7
+
+1. Open `/etc/httpd/conf/httpd.conf` file for editing:
+
+   ```bash
+   vim /etc/httpd/conf/httpd.conf`
+   ```
+
+1. Locate the block that starts with:
+
+   ```conf
+   <Directory /var/www/html>
+   ```
+
+1. Change the value of `AllowOverride` to `<value from Apache site>`.
+
+   For example,
+
+   ```conf
+   <Directory /var/www/>
+     ...
+     Options Indexes FollowSymLinks MultiViews
+     AllowOverride <value from Apache site>
+     Order allow,deny
+     Allow from all
+     ...
+   <Directory>
+   ```
+
+   {:.bs-callout-info}
+   The preceding values for `Order` might not work in all cases. For more information, see the Apache documentation ([2.4](https://httpd.apache.org/docs/2.4/mod/mod_authz_host.html#order)).
 
-### Enable rewrites and .htaccess for Apache 2.2 (including CentOS 7)
+1. Save the file and exit the text editor.
 
-{% include install/allowoverrides22.md %}
+1. To apply Apache settings, restart Apache.
+
+   ```bash
+   service apache2 restart
+   ```
+
+{:.bs-callout-info}
+Failure to enable these settings typically results in no styles displaying on your storefront or Admin.
+
+### Enable rewrites and .htaccess for Ubuntu
+
+1. Open `/etc/apache2/sites-available/default` file for editing:
+
+   ```bash
+   vim /etc/apache2/sites-available/default
+   ```
+
+1. Locate the block that starts with:
+
+   *  Ubuntu 12: `<Directory /var/www/>`
+   *  Ubuntu 14: `<Directory /var/www/html>`
+
+1. Change the value of `AllowOverride` to `<value from Apache site>`.
+
+   An example for Ubuntu 12 follows:
+
+   ```conf
+   <Directory /var/www/>
```

Comment:
Could I ask you to preserve the same indentation we have here, please? 

https://github.com/magento/devdocs/pull/6758/files#diff-4d1597de1c3c126a322fcf20ab9d8e48R214

So we have 

```
<Directory /var/www/>
  Options Indexes FollowSymLinks MultiViews
  AllowOverride <value from Apache site>
  ...
</Directory>
```

Thank you!

Reply:
@rogyar , thank you for your notice

**fixed**